uint16 address
bool[] bits
uint8 slave
---
